src/openms/include/OpenMS/CONCEPT/Constants.h

```diff
@@ -441,6 +441,42 @@ namespace OpenMS
               String
       */
       inline const std::string   XFDR_FDR = "XFDR:FDR";
+
+      /** User parameter name for best ion annotation in a ConsensusFeature, taken from the best quality feature. (Required for IIMN)
+              String
+      */
+      inline const std::string   IIMN_BEST_ION = "best ion";
```

Comment:
did anyone wonder why do we use strings?
stringview would be sufficient?

Reply:
We may need a stringview overload to look up metavalues .... maybe thats why :P

Reply:
How does this make a difference? There would still be string comparisons upon lookup?

Reply:
true, but no construction cost (at least the rhs would be constexpr automatically for SV's) and no heap allocation for the string if SSO does not kick in (better locality).